from restclients_core.exceptions import DataFailureException
